maven_iuvs/echelle.py

```diff
@@ -1542,17 +1546,40 @@ def flatten(light_fits, processed_data):
 
     return spec, unc
 
-
-def fit_flat_data(light_fits, spectrum, data_unc, bad_frames=None, 
-                  calibration="new", return_each_line_fit=False, ints_to_fit=1,  
+_fit_parameter_names = ['total_brightness_H',  # DN
+                        'total_brightness_D',  # DN
+                        'total_brightness_IPH',  # DN
+                        'central_wavelength_H',  # nm
+                        'central_wavelength_IPH',  # nm  
+                        'width_IPH',  # nm
+                        'background_m',
+                        'background_b']
+_unc_parameter_names = ['unc_total_brightness_H', 
```

Comment:
should probably use a list comprehension here for safety: `['unc_'+pname for pname in _fit_parameter_names]`

Reply:
Fixed in 38c81f7ccf7737d2e3e3e95e56dc52065f78456a